src/api/kube.ts

```diff
@@ -1640,7 +1640,35 @@ export class KubeHelper {
       return crs[0]
     } catch (e) {
       if (e.response.statusCode === 404) {
-        // There is no CR 'checluster`
+          // There is no CR 'checluster`
+        return
+      }
+      throw this.wrapK8sClientError(e)
+    }
+  }
+
+  /**
+   * Returns `che.eclipse.org/v1alpha1' in the given namespace.
+   */
+  async getCheManagerInstance(namespace: string): Promise<any | undefined> {
+    const customObjectsApi = this.kubeConfig.makeApiClient(CustomObjectsApi)
+    try {
+      const { body } = await customObjectsApi.listNamespacedCustomObject('che.eclipse.org', 'v1alpha1', namespace, 'chemanagers')
+      if (!(body as any).items) {
+        return
+      }
+
+      const crs = (body as any).items as any[]
+      if (crs.length === 0) {
+        return
+      } else if (crs.length !== 1) {
+        throw new Error(`Too many resources '${CHE_CLUSTER_CRD}' found in the namespace '${namespace}'`)
```

Comment:
is it really `${CHE_CLUSTER_CRD}` ? copy/paste?

Reply:
My mistake